tests/test_subdirectory.py

```diff
@@ -0,0 +1,36 @@
+from plumbum import local
+from plumbum.cmd import git
+
+import copier
+from copier.config import make_config
+from copier.main import update_diff
+
+
+def test_copy_subdirectory_api_option(dst):
```

Comment:
I plan to deprectate `dst` as pylint has a valid alternative fixture upstream:

```suggestion
def test_copy_subdirectory_api_option(tmp_path):
```

The same in the rest of your test, please.